lib/drivers/openai.js

```diff
@@ -0,0 +1,56 @@
+const MAX_PROMPT_LEN = 10000
```

Comment:
MAX_PROPMPT_LEN variable is duplicated, if you want we can extract this into separate enum